### DISCUSSION TITLE
How can I zoom/scale the UI?  I have a 4K monitor and the text is super small

### DISCUSSION THREAD

**Discussion by skostojohn** [alainm23/planify#1308] [Q&A]
Thanks in advance... I tried Ctrl++, Ctrl+-, preferences, etc.  

Scott

**Comment by adriandelgg** [comment]
I need an answer to this as well. So annoying

**Comment by adriandelgg** [comment]
Did you figure it out? @skostojohn

**Reply by skostojohn** [reply]
I did not.

**Comment by Riksu9000** [comment]
In Gnome you can use fractional scaling or enable large text in the accessibility settings. The text size in planify is consistent with other gnome apps.

**Comment by sojek4** [comment]
Have you tired this one?

`GDK_SCALE=2 flatpak run io.github.alainm23.planify`
